modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java

```diff
@@ -100,15 +106,24 @@ public String getCmd() {
 			_portletRequest, "customElementURLs", customElementURLs);
 
 		if (customElementURLs.length == 0) {
-			customElementURLs = new String[1];
+			customElementURLs = new String[] {
+				"https://liferay.github.io/liferay-frontend-projects" +
```

Comment:
you are going to send it before it's enabled :O

Reply:
Yes. I said it in the PR description.

Reply:
See https://github.com/liferay-frontend/liferay-portal/pull/1654#issuecomment-966316823